benchmarks/dynamo/ci_expected_accuracy/update_expected.py

```diff
@@ -42,15 +73,17 @@
 
 def query_job_sha(repo, sha):
     params = {
-        "parameters": [
-            {"name": "sha", "type": "string", "value": sha},
-            {"name": "repo", "type": "string", "value": repo},
-        ]
+        "queryVariables": {"sha": sha, "repo": repo},
     }
 
-    r = requests.post(url=ARTIFACTS_QUERY_URL, json=params)
-    data = r.json()
-    return data["results"]
+    r = requests.post(
+        url=ARTIFACTS_QUERY_URL,
+        data=json.dumps(params),
+        headers={"Content-Type": "application/json"},
+        # This key can only access query endpoints, specifically the url above at the moment
+        auth=("eJBpUENTBQ4MdbHPYxrl", "4b1draOBY5zwva7QXTBgAwqbYrpNKFaXGNYqASt5a9"),
```

Comment:
I guess this is a read-only key, but I think we should still keep it a secret as this looks like an open invitation to people trying to gain access or ddos our query endpoint

Reply:
How do you propose distributing the key to everyone who wants to run this script?

Reply:
I think we should make a request to CH to see if they allow no-auth read-only request to their API endpoint like what Rockset gives us.  Another solution that doesn't block us is to create a simple API on HUD that forward the queries to CH and returns the results, we can then keep the read-only secret on Vercel

Reply:
Don't these solutions have the same problem as before?

Reply:
Removed the credentials an put a link to them instead